spec/views/entities/new.html.erb_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'entities/new.html.erb', type: :feature do
+  let(:user) { User.create(name: 'Yemi', email: 'test@test.com', password: 'password') }
+  describe 'Testing createing new entity' do
+    before :each do
+      sign_in user
+      visit new_entity_path
+    end
+
+    context 'When creating a new entity' do
+      it 'should see the entity name' do
+        fill_in 'Name', with: 'Execise'
+        fill_in 'Amount', with: 20
+        click_button 'Submit'
```

Comment:
- [ ] Kindly make sure all your tests are passing, one of the tests is not passing 🙃 

<img width="981" alt="Screenshot 2023-10-21 at 19 52 56" src="https://github.com/yemidada/MyBudget/assets/74501523/308f2f36-9556-4d60-9ae8-b21f8c32025c">